src/Utilities/StdArrayHelpers.hpp

```diff
@@ -188,3 +188,59 @@ auto map_array(const std::array<T, Dim>& array, const F& f) noexcept {
   return std_array_helpers_detail::map_array_impl(
       array, f, std::make_index_sequence<Dim>{});
 }
+
+
+/**\ingroup UtilitiesGroup
+ * \brief Declares a binary function on an array, intended for operations
+ *
+ * \param RESULT_TYPE the array type that is the result of the operation (e.g A
```

Comment:
array type -> `value_type` (with the ticks around it for code format). The STL has  member type aliases `value_type` in all containers, and for your example it is `A`